router/pump.go

```diff
@@ -89,6 +89,15 @@ func (p *LogsPump) Setup() error {
 	return nil
 }
 
+func (p *LogsPump) rename(event *docker.APIEvents) {
+    p.mu.Lock()
+    defer p.mu.Unlock()
+    container, err := p.client.InspectContainer(event.ID)
```

Comment:
Ah I misread that then. Well if there's no compatibility issues then I'm happy to update this PR.